planning/behavior_path_planner/src/utilities.cpp

```diff
@@ -1227,7 +1227,7 @@ OccupancyGrid generateDrivableArea(
     }
 
     // Closing
-    constexpr int num_iter = 10;  // TODO(Horibe) Think later.
+    constexpr int num_iter = 1;  // TODO(Horibe) Think later.
     cv::Mat cv_erode, cv_dilate;
     cv::erode(cv_image, cv_erode, cv::Mat(), cv::Point(-1, -1), num_iter);
```

Comment:
@takayuki5168 
[FYI] you can replace erode and dilate processing to morphologyEx instead if you want to
[cv::morphologyEx](https://docs.opencv.org/3.4/d4/d86/group__imgproc__filter.html#ga67493776e3ad1a3df63883829375201f)

Reply:
Thank you. In this PR, to be simple, I will keep the current change.